docs/features/enterprise/tekton.md

```diff
@@ -0,0 +1,71 @@
+# Tekton 
+
+*Since version  v.2.12.0
+
+Tekton is a Kubernetes extension that installs and runs on your Kubernetes cluster. It defines a set of Kubernetes Custom Resources that act as building blocks from which you can assemble CI/CD pipelines. Once installed, Tekton Pipelines becomes available via the Kubernetes CLI (kubectl) and via API calls, just like pods and other resources. For more information about TektonCD features check out the [official documentation](https://github.com/tektoncd/pipeline/tree/master/docs).
+
+**Setting up Tekton**
+
+In order to set up TektonCD to work with Kubernetic you, firstly, need to install the Operation Lifecycle Manager (OLM). See [Installing OLM](https://docs.kubernetic.com/#/features/settings/addons?id=installing-the-operator-lifecycle-manager-olm).
+
+After you have the OLM installed, go to the "Available Operators" tab, select "tekton-operator" and press "Install"
+
+![](../images/tekton-operator.png)
```

Comment:
There is no image shown for tekton-operator (https://deploy-preview-219--docs-kubernetic.netlify.app/#/features/enterprise/tekton)

Is the tekton operator available now on OLM? Last time I checked it was not, and installation is done outside of OLM process.

Reply:
ok, tekton-operator is live https://operatorhub.io/operator/tektoncd-operator